apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/Vedtakstidslinje.kt

```diff
@@ -12,20 +12,26 @@ class Vedtakstidslinje(private val vedtak: List<Vedtak>) {
     private val iverksatteVedtak = hentIverksatteVedtak()
 
     fun erLoependePaa(dato: LocalDate): LoependeYtelse {
-        if (iverksatteVedtak.isEmpty()) return LoependeYtelse(false, dato)
+        val erUnderSamrodning = vedtak.any { listOf(VedtakStatus.TIL_SAMORDNING, VedtakStatus.SAMORDNET).contains(it.status) }
```

Comment:
Vi bør egentlig kunne klare oss uten SAMORDNET. Hvis vi har noen vedtak i den statusen når reguleringen kommer så er det noe teknisk feil et sted som vi bør kunne ordne opp i forkant. Feilen er da et eller annet sted i utbetalingsløypa.

Reply:
Skal jeg fjerne SAMORDNET helt også stoler vi på at vi alltid vil sjekke det i forveien? Eller skal vi ha beholde den slik at den feiler om vi skulle glemme eller overse en i den manuelle rutina?

Reply:
Stemmer for at vi beholder den

Reply:
Behold, manuelle rutinar kan gleppe.